apidAnalytics_suite_test.go

```diff
@@ -220,7 +217,7 @@ func insertTestData(db apid.DB) {
 
 	txn, err := db.Begin()
 	Expect(err).ShouldNot(HaveOccurred())
-	txn.Exec("INSERT INTO APP_CREDENTIAL_APIPRODUCT_MAPPER (tenant_id,"+
+	txn.Exec("INSERT INTO kms_APP_CREDENTIAL_APIPRODUCT_MAPPER (tenant_id,"+
```

Comment:
lol. Not sure you need to camel case here. Kind of looks odd. But otherwise look good.
I assume you have tested to ensure there are no regressions.